packages/cli/src/WebhookHelpers.ts

```diff
@@ -213,8 +217,22 @@ export async function executeWebhook(
 		throw new ResponseHelper.ResponseError(errorMessage, 500, 500);
 	}
 
+	let user: N8nUserData;
+	if (
+		(workflowData as WorkflowEntity).shared?.length &&
+		(workflowData as WorkflowEntity).shared[0].user
```

Comment:
One day we need to refactor to ensure we are always working with the model  `WorkflowEntity` and not with a slightly adjusted version `IWorkflowDb`.